src/shieldtest.js

```diff
@@ -0,0 +1,233 @@
+"use strict";
+
+import { map } from "./americana.js";
+import * as shield from "./js/shield.js";
+import { shields } from "./js/shield_defs.js";
+import * as gfx from "./js/screen_gfx.js";
+
+const once = (emitter, name, { signal } = {}) =>
+  new Promise((resolve, reject) => {
+    emitter.once(name, resolve);
+    signal?.addEventListener("abort", reject);
+  });
+
+await once(map, "load");
+
+let networks = [
+  "default",
+
+  // Circles, pills, ovals
+  "US:DE",
+  "KR:national",
+  "US:IA",
+  "US:NM",
+  "US:KS",
+
+  // Rectangles
+  "US:CT",
+  "US:AR:Lee",
+  "US:AR:Baxter",
+  "US:WY",
+  "CA:NS:R",
+  "CA:YT",
+
+  // Basic Angular Shapes
+  "CA:ON:secondary",
+  "US:NE",
+  "US:MN:Aitkin:Park_Access",
+  "US:PA",
+  "US:PA:Turnpike",
+
+  "CA:AB:primary",
+  "CA:SK:tertiary",
+  "CA:SK:primary",
+  "PH:N",
+  "PH:E",
+  "US:TX:Montgomery:MCTRA",
+
+  "US:TN:secondary",
+  "US:NC",
+
+  "US:IA:CR",
+  "US:CO:Douglas",
+  "US:MN:Aitkin:CSAH",
+  "US:TX:Harris:HCTRA",
+
+  "JP:prefectural",
+
+  "my:federal",
+  "TR:motorway",
+  "US:NY",
+  "US:NY:Parkway",
+  "NL:S:Amsterdam",
```

Comment:
Non dimenticare gli ottagoni!
```suggestion
  "NL:S:Amsterdam",
  "IT:A-road",
```